### PR TITLE
Explicit padding for ConformerConvolutionV1

### PR DESCRIPTION
**Pull request by JackTemaki**
In case we want to use the model in pipelines for ONNX export, we can not use `padding="same"` for now:
https://github.com/pytorch/pytorch/issues/68880

While workarounds would be possible, I would strongly advise to not introduce export hacks already for our first model. Also while export might work, it is unclear if the ONNX runtime does.

### CODE REVIEW

**Review by mmz33 — APPROVED** [review]

**Review by vieting — APPROVED** [review]

File: i6_models/parts/conformer/convolution.py [hunk 1]
```diff
@@ -28,20 +28,23 @@ class ConformerConvolutionV1(nn.Module):
...
+    https://github.com/pytorch/pytorch/issues/68880
     """
 
     def __init__(self, model_cfg: ConformerConvolutionV1Config):
         """
         :param model_cfg: model configuration for this module
         """
         super().__init__()
-
+        assert model_cfg.kernel_size % 2 == 1, "ConformerConvolutionV1 only supports odd kernel sizes"
```
Comment (albertz) [inline i6_models/parts/conformer/convolution.py:41]: I thought the decision was that this check is in the cfg and not here? (Even though I disagreed with this decision.)

Comment (JackTemaki) [reply i6_models/parts/conformer/convolution.py]: Good point, why not something like this now?

File: i6_models/parts/conformer/convolution.py [hunk 2]
```diff
@@ -23,25 +23,35 @@ class ConformerConvolutionV1Config(ModelConfiguration):
...
+    https://github.com/pytorch/pytorch/issues/68880
     """
 
     def __init__(self, model_cfg: ConformerConvolutionV1Config):
         """
         :param model_cfg: model configuration for this module
         """
         super().__init__()
-
+        model_cfg.check_valid()
```
Comment (Atticus1806) [inline i6_models/parts/conformer/convolution.py:48]: Now you call `check_valid()` twice, is this really required?

Comment (JackTemaki) [reply i6_models/parts/conformer/convolution.py:48]: This seems unavoidable if we want to have a check at creation but also a final check at runtime.

**Review by Atticus1806 — APPROVED** [review]
